Makefile

```diff
@@ -59,8 +59,21 @@ lint:
 	@echo "checking lint"
 	@./.tool/lint
 
-test:
-	go test -v -race -cover $(shell go list ./... | grep -v /vendor/)
+.PHONY: .gofmt .govet .gotest
+
+PACKAGES = $(shell go list ./... | grep -v /vendor/)
+test: .gofmt .govet .gotest
+
+FILES = $(shell find ./ -name *.go | grep -v vendor)
+.gofmt:
+	OUT=$$(gofmt -s -d $(FILES)); if test -n "$${OUT}"; then echo "$${OUT}" && exit 1; fi
+
+.govet:
+	go vet -x $(PACKAGES)
+
+UTDIRS = ./image/...
+.gotest:
+	go test -v -race -cover $(UTDIRS)
```

Comment:
why `go test ` only cover `UTDIRS = ./image/...` ? the old way is to cover `$(shell go list ./... | grep -v /vendor/)`?

Reply:
>?   	github.com/opencontainers/image-tools/cmd/oci-image-tool	[no test files]
ok  	github.com/opencontainers/image-tools/image	1.120s	coverage: 56.5% of statements
?   	github.com/opencontainers/image-tools/version	[no test files]

Because only the image has the test file, this can improve efficiency.

Reply:
But I think it's better to keep the old way, because we can't make sure we will not add test file in other dir in future.

Reply:
Fine, updated.